Source/WebCore/rendering/style/StyleGradientImage.cpp

```diff
@@ -1124,55 +1277,50 @@ Ref<Gradient> StyleGradientImage::createGradient(const DeprecatedLinearData& lin
 
 // MARK: - Radial create.
 
-Ref<Gradient> StyleGradientImage::createGradient(const RadialData& radial, const RenderElement& renderer, const FloatSize& size, const RenderStyle& style) const
+Ref<Gradient> StyleGradientImage::createGradient(const RadialData& radial, const RenderElement&, const FloatSize& size, const RenderStyle& style) const
```

Comment:
Can we eliminate the `renderer` argument entirely instead of passing it and ignoring it?